client/scala/src/main/scala/ai/verta/blobs/dataset/PathBlob.scala

```diff
@@ -60,7 +78,10 @@ object PathBlob {
    */
   def reduce(firstBlob: PathBlob, secondBlob: PathBlob): Try[PathBlob] = {
     if (firstBlob.notConflicts(secondBlob))
-      Success(new PathBlob(firstBlob.contents ++ secondBlob.contents))
+      Success(new PathBlob(
+        firstBlob.contents ++ secondBlob.contents,
+        firstBlob.enableMDBVersioning && secondBlob.enableMDBVersioning
```

Comment:
Maybe cause an error if they're different? Silencing such an important and explicit request from the user seems bad.

Reply:
Fixed + added a test that demonstrates the new expected behavior.